web/src/dropdown_widget.ts

```diff
@@ -90,7 +137,7 @@ export class DropdownWidget {
             `${this.widget_id}, ${this.widget_wrapper_id}`,
             (e) => {
                 if (e.key === "Enter") {
-                    $(`${this.widget_id}`).trigger("click");
+                    $(this.widget_id).trigger("click");
```

Comment:
This should have been named `widget_selector`, not `widget_id`. `#foo` is a selector for the ID `foo`.

Reply:
Okay, I will add a commit for later for this.

Reply:
Done.